File: Habitica/src/main/java/com/habitrpg/android/habitica/widget/TodoListWidgetProvider.java
package com.habitrpg.android.habitica.widget;

import com.habitrpg.android.habitica.R;

public class TodoListWidgetProvider extends TaskListWidgetProvider{

    @Override
    protected Class getServiceClass() {
        return TodosWidgetService.class;
    }

    @Override
    protected Class getProviderClass() {
        return TodoListWidgetProvider.class;
    }

    @Override
    protected int getTitleResId() {
        return R.string.todos;
    }
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/widget/TodoListWidgetProvider.kt
package com.habitrpg.android.habitica.widget

import androidx.glance.appwidget.GlanceAppWidget
import com.habitrpg.android.habitica.widget.glance.widgets.TodoTaskListGlanceWidget

class TodoListWidgetProvider : HydratingGlanceReceiver() {
    override val glanceAppWidget: GlanceAppWidget = TodoTaskListGlanceWidget()
}
